### PR TITLE
add iperf example

### PR DESCRIPTION
**Pull request by ajssmith**

### CODE REVIEW

File: network-iperf/README.md [hunk 1]
```diff
@@ -0,0 +1,137 @@
+# Testing network throughput across clusters
+
+This tutorial demonstrates how to perform real time network throughput measurements on an application router network using the iperf3 tool.
+
+In this tutorial, you will deploy iperf3 servers in separate clusters. You will also create an application router network, which enable the iperf3 instances to run in client mode and access peer iperf3 servers running on the different clusters (e.g. private and public).
```
Comment (bhardesty) [inline network-iperf/README.md]: ...which _will_ enable...

File: network-iperf/README.md [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+3. In the terminal for the second public cloud, deploy the third iperf3 server:
+
+   ```bash
+   $ cd ~/network-iperf-demo/demos/network-iperf/
+   $ oc apply -f deployment-iperf3-c.yaml
+   ```
+
+## Step 4: Run benchmark tests across the clusters
+
+After deploying the iperf3 servers into the private and publicd cloud clusters, the application router network connects the servers and enables communications even thought the are running in separate clusters.
```
Comment (bhardesty) [inline network-iperf/README.md]: thought -> though

File: network-iperf/README.md [hunk 3]
```diff
@@ -0,0 +1,137 @@
+# Testing network throughput across clusters
+
+This tutorial demonstrates how to perform real time network throughput measurements on an application router network using the iperf3 tool.
```
Comment (bhardesty) [inline network-iperf/README.md]: real-time

**Comment by bhardesty** [comment]
@ajssmith what is the purpose of this demo - is it something that a user would run on its own, or is it something that you would most likely run after completing one of the other demos (as in, "I've completed the MongoDB demo, but now I'd like to test the performance")?

If it's the latter, we might want to simplify the readme - the application router network could be a prereq, and the tutorial could start with the iperf3 setup.

**Comment by ajssmith** [comment]
Thanks for the review comments. This demo is intended to run on its own. It is likely to be an operational practice to test and measure communication performance across clusters. I would expect we would eventually add a few more performance tool examples for tcp, http and udp but this was a start.

**Pull request merged**